meilisearch-http/src/main.rs

```diff
@@ -50,25 +55,14 @@ async fn main() -> Result<(), MainError> {
         _ => unreachable!(),
     }
 
-    //if let Some(path) = &opt.import_snapshot {
-    //snapshot::load_snapshot(&opt.db_path, path, opt.ignore_snapshot_if_db_exists, opt.ignore_missing_snapshot)?;
-    //}
-
     let data = Data::new(opt.clone())?;
 
-    //if !opt.no_analytics {
-    //let analytics_data = data.clone();
-    //let analytics_opt = opt.clone();
-    //thread::spawn(move || analytics::analytics_sender(analytics_data, analytics_opt));
-    //}
-
-    //if let Some(path) = &opt.import_dump {
-    //dump::import_dump(&data, path, opt.dump_batch_size)?;
-    //}
-
-    //if opt.schedule_snapshot {
-    //snapshot::schedule_snapshot(data.clone(), &opt.snapshot_dir, opt.snapshot_interval_sec.unwrap_or(86400))?;
-    //}
+    #[cfg(feature = "analytics")]
+    if !opt.no_analytics {
+        let analytics_data = data.clone();
+        let analytics_opt = opt.clone();
+        tokio::task::spawn(analytics::analytics_sender(analytics_data, analytics_opt));
+    }
```

Comment:
Maybe we should prevent data to be sent in debug builds, just like above

Reply:
This was not the case in the current Meilisearch. But if @qdequele or @gmourier are ok with that, I can do it :slightly_smiling_face:

Reply:
To be sure, are we talking about a MeiliSearch compiled by the user in debug mode and not in release mode? I ask the question to be sure that it does not concern the log level. 

If this is the case, I don't see any immediate need to have analytics on the uses of Meilisearch when it is compiled in debug. It also avoids polluting Amplitude with data that doesn't represent the majority of our users.

Reply:
Yes, you understood perfectly. Right now Meilisearch (and this pr) are sending amplitude analytics even when compiled in Debug mode. Sentry is not sending any analytics while in debug mode though.

Basically, the debug mode is faster to compile, slower to execute and obviously easier to debug.
Every binary we ship is compiled in release mode so someone using a debug build build it himself from the source.

Reply:
The spec has been updated to mention the debug build and state we don't collect any telemetry in debug build!